### ISSUE TITLE
Unstoppable animations are an accessibility failure

### ISSUE THREAD

**Issue by adamshostack** [VSCodium/vscodium#2033]
Type: <b>Bug</b>

Reproduction steps: Read the release notes. Get to "Copy settings URL" See animation. Be distracted by animation moving things on screen. Try to make it stop. Be unable to make it stop because there's no "Stop motion" affordance, even after clicking the videos. In other places, the affordance is only shown(!) after you click the video.  Some, but not all of the animations are videos and can be stopped (with effort). Others are not.

Impact: I cannot read the full release notes because the motion keeps taking my attention.

Suggested resolution:
1. All animations should be videos with the ability to stop them from playing.
2. All animations should only play after user interaction.
3. If you choose to auto-play interactions, the "Stop" affordance should be visible by default.

Background/context: Motion is distracting to a large segment of the population. That's why Apple's IOS (for example) has "reduce motion" under accessibility. I literally can't read what I'm writing with the animation going, it keeps grabbing my attention.

VSCodium version: VSCodium 1.93.0 (44560ce62f89cd33130676a7c21c22b476173a94, 2024-09-09T13:48:06.715Z)
OS version: Darwin x64 23.6.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-9600K CPU @ 3.70GHz (6 x 3700)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|4, 3, 3|
|Memory (System)|40.00GB (0.01GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (2)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-codeql|Git|1.14.0
markdown-pdf|yza|1.5.0

</details>
